src/runtime/types.ts

```diff
@@ -601,3 +601,14 @@ export interface ModuleOptionsNormalized extends ModuleOptions {
     fullBaseUrl: string
   }
 }
+export interface SessionCookie {
+  lastRefreshedAt?: SessionLastRefreshedAt
+  data?: SessionDataObject
+}
+
+// Augment types
+declare module 'nuxt/schema' {
+  interface PublicRuntimeConfig {
+    auth: ModuleOptionsNormalized
+  }
+}
```

Comment:
Question as to why was this added?

Reply:
Sorry, I don't remember, but I guess it was causing merge conflicts.